msgraph/remote.go

```diff
@@ -51,7 +52,9 @@ func (r *impl) MakeClient(ctx context.Context, token *oauth2.Token) remote.Clien
 
 // MakeSuperuserClient creates a new client used for app-only permissions.
 func (r *impl) MakeSuperuserClient(ctx context.Context) (remote.Client, error) {
-	httpClient := &http.Client{}
+	httpClient := &http.Client{
+		Timeout: time.Second * 60,
```

Comment:
60 seconds seems like a lot. What was the rationale here?

Reply:
@wiggin77 Can you please look into the above?

Reply:
I see in your description for the PR you reference https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/httpclient/httpclient-timeout-method#remarks but that seems irrelevant here.

The default timeout for https://pkg.go.dev/net/http#Client is actually 0 (which means no timeout)

```
package main

import (
	"fmt"
	"net/http"
)

func main() {
	client := http.Client{}

	fmt.Println(client.Timeout)
}
```

Output:
```
0s
```

Reply:
If we don't have any data to help our decision, 10s would probably be safe.

Reply:
5s in my experience is too short and leads to spurious errors.  I chose 60 seconds as balance between handling spurious interruptions and not piling up connections.  This is not a section of code that will generate a ton of concurrent connections.